src/markdown-pages/automate-workflows/automated-tagging.mdx

```diff
@@ -122,44 +169,38 @@ These section describe, to the deployer, the actions to take when it executes. F
       "deploy_script_path": "deploy/linux/roles",
       "port": 5001,
       "destinations": ["host1"],
-      "files":[{
-        "destination_filepath": "engine/data/index.json",
-        "content": [
-          "<h1>Welcome!</h1>",
-          "Congrats! You have successfully setup our <b>Hello World</b> demo environment. So, what's been deployed?",
-          "<ul><li>A simple host on your cloud provider, e.g. AWS, Azure.</li>",
-          "<li>A simple node application that is serving up this page.</li>",
-          "<li>New Relic instrumentation, specifically, the infrastructure agent and the node agent.</li></ul>",
-          " ",
-          "Next, start generating some traffic on the app. This can be done by clicking the <b>Inventory</b> button, which will simply return a collection of items. You can then click each item to see its details.",
-          "You can also use postman, or a similar tool to generate a burst of requests. The APIs for the app are quite simple:",
-          "<ul><li>Inventory <a href=\"[service:app1]/api/inventory\" target=\"_blank\">[service:app1]/api/inventory</a></li>",
-          "<li>Item details <a href=\"[service:app1]/api/inventory/1\" target=\"_blank\">[service:app1]/api/inventory/[item number]</a></li></ul>",
-          " ",
-          "Now, head on over to New Relic and explore what's been created.",
-          "<ul><li>Search for your app and host using <a href=\"https://one.newrelic.com/redirect/search/Hello\" target=\"_blank\">global search</a>.</li>",
-          "<li>Check out <a href=\"https://one.newrelic.com/redirect/explorer\" target=\"_blank\">Entity Explorer</a>.</li>",
-          "<li>Build some awesome graphs in Chart Builder with this NRQL:</li></ul>",
-          "** SELECT count(*) FROM Transaction FACET request.headers.userAgent WHERE appName = 'Hello-App1' LIMIT MAX",
-          "** SELECT average(duration) FROM Transaction FACET name WHERE appName = 'Hello-App1'",
-          "** SELECT count(*) FROM Transaction TIMESERIES FACET httpResponseCode WHERE appName = 'Hello-App1'",
-          "** SELECT average(memoryUsedPercent) as '% Used', average(memoryFreePercent) as '% Free' FROM SystemSample TIMESERIES WHERE apmApplicationNames = '|Hello-App1|'",
-          " ",
-          "That's it! We hope you enjoyed this quick <b>Hello World</b> example. Please give us feedback and ideas in our [Explorer's Hub community](https://discuss.newrelic.com/c/build-on-new-relic).",
-          " ",
-          "<h2>Remember:</h2>Make sure to tear down the infrastructure and app you just provisioned!"
-        ]
-      }]
+      "files": [
+        {
+          "destination_filepath": "engine/data/index.json",
+          "content": <some content>
```

Comment:
This isn't technically valid JSON with `<some content>`, but I felt that including all the HTML was unnecessary. Removing it helped to focus more on what the section is really about. Thoughts?

Reply:
good question. since this is for reference and not for copy/paste purposes, I would just add a note that some HTML has been removed for clarity.